From cc81221f4b702c086eda5abac4033274ad52de4e Mon Sep 17 00:00:00 2001
From: Maxim Yu. Osipov <mosipov@dev.rtsoft.ru>
Date: Wed, 29 May 2013 11:23:49 +0200
Subject: [PATCH 117/127] Revert "arm: tegra: ktt30: Added support of case
 open."

Migrated to different scheme of intrusion detection described
in Documentaion/hwmon/sysfs-interface. It's used by KEAPI.

This reverts commit 8db74fe6e63c4d0a5aeb1abe40c542ad7b9758fe.

Signed-off-by: Maxim Yu. Osipov <mosipov@dev.rtsoft.ru>
Signed-off-by: Valentine Barshak <vbarshak@dev.rtsoft.ru>
---
 arch/arm/mach-tegra/board-ktt30.c |   28 ++++++++--------------------
 1 files changed, 8 insertions(+), 20 deletions(-)

diff --git a/arch/arm/mach-tegra/board-ktt30.c b/arch/arm/mach-tegra/board-ktt30.c
index 32072fc..473a664 100644
--- a/arch/arm/mach-tegra/board-ktt30.c
+++ b/arch/arm/mach-tegra/board-ktt30.c
@@ -74,17 +74,6 @@
 #include "pm.h"
 #include "wdt-recovery.h"
 
-#define GPIO_SW_KEY(_id, _gpio, _iswake)	\
-	{					\
-		.code = _id,			\
-		.gpio = _gpio,			\
-		.active_low = 1,		\
-		.desc = #_id,			\
-		.type = EV_SW,			\
-		.wakeup = _iswake,		\
-		.debounce_interval = 1,		\
-	}
-
 #define GPIO_IKEY(_id, _irq, _iswake, _deb)	\
 	{					\
 		.code = _id,			\
@@ -620,28 +609,27 @@ static struct platform_device ktt30_audio_wm8903_device = {
 	},
 };
 
-static struct gpio_keys_button ktt30_keys[] = {
+static struct gpio_keys_button ktt30_int_keys[] = {
 	[0] = GPIO_IKEY(KEY_POWER,
-		 TPS6591X_IRQ_BASE + TPS6591X_INT_PWRON, 1, 100),
-	[1] = GPIO_SW_KEY(SW_LID, TPS6591X_GPIO_5, 0),
+		 TPS6591X_IRQ_BASE + TPS6591X_INT_PWRON, 1, 100)
 };
 
-static struct gpio_keys_platform_data ktt30_keys_pdata = {
-	.buttons	= ktt30_keys,
-	.nbuttons       = ARRAY_SIZE(ktt30_keys),
+static struct gpio_keys_platform_data ktt30_int_keys_pdata = {
+	.buttons	= ktt30_int_keys,
+	.nbuttons       = ARRAY_SIZE(ktt30_int_keys),
 };
 
-static struct platform_device ktt30_keys_device = {
+static struct platform_device ktt30_int_keys_device = {
 	.name   = "gpio-keys",
 	.id     = 0,
 	.dev    = {
-		.platform_data  = &ktt30_keys_pdata,
+		.platform_data  = &ktt30_int_keys_pdata,
 	},
 };
 
 int __init ktt30_keys_init(void)
 {
-	platform_device_register(&ktt30_keys_device);
+	platform_device_register(&ktt30_int_keys_device);
 	return 0;
 }
 
-- 
1.7.4.4

